### PR TITLE
HBASE-23788 ROW_INDEX_V1 encoder should consider the secondary index size with the encoded data size tracking

### PR DESCRIPTION
**Pull request by abhinabasarkar**

### CODE REVIEW

**Comment by Apache-HBase** [comment]
:broken_heart: **-1 overall**






| Vote | Subsystem | Runtime | Comment |
|:----:|----------:|--------:|:--------|
| +0 :ok: |  reexec  |   1m 25s |  Docker mode activated.  |
||| _ Prechecks _ |
| +1 :green_heart: |  dupname  |   0m  0s |  No case conflicting files found.  |
| +1 :green_heart: |  hbaseanti  |   0m  0s |  Patch does not have any anti-patterns.  |
| +1 :green_heart: |  @author  |   0m  0s |  The patch does not contain any @author tags.  |
| +1 :green_heart: |  test4tests  |   0m  0s |  The patch appears to include 1 new or modified test files.  |
||| _ master Compile Tests _ |
| +0 :ok: |  mvndep  |   0m 34s |  Maven dependency ordering for branch  |
| +1 :green_heart: |  mvninstall  |   5m 59s |  master passed  |
| +1 :green_heart: |  compile  |   1m 25s |  master passed  |
| +1 :green_heart: |  checkstyle  |   1m 38s |  master passed  |
| +1 :green_heart: |  shadedjars  |   5m 19s |  branch has no errors when building our shaded downstream artifacts.  |
| +1 :green_heart: |  javadoc  |   0m 58s |  master passed  |
| +0 :ok: |  spotbugs  |   4m 56s |  Used deprecated FindBugs config; considering switching to SpotBugs.  |
| +1 :green_heart: |  findbugs  |   5m 48s |  master passed  |
||| _ Patch Compile Tests _ |
| +0 :ok: |  mvndep  |   0m 13s |  Maven dependency ordering for patch  |
| +1 :green_heart: |  mvninstall  |   5m 43s |  the patch passed  |
| +1 :green_heart: |  compile  |   1m 29s |  the patch passed  |
| +1 :green_heart: |  javac  |   1m 29s |  the patch passed  |
| +1 :green_heart: |  checkstyle  |   1m 38s |  the patch passed  |
| +1 :green_heart: |  whitespace  |   0m  0s |  The patch has no whitespace issues.  |
| -1 :x: |  shadedjars  |   4m  6s |  patch has 10 errors when building our shaded downstream artifacts.  |
| +1 :green_heart: |  hadoopcheck  |  17m 38s |  Patch does not cause any errors with Hadoop 2.8.5 2.9.2 or 3.1.2.  |
| -1 :x: |  javadoc  |   0m 21s |  hbase-common generated 1 new + 0 unchanged - 0 fixed = 1 total (was 0)  |
| -1 :x: |  findbugs  |   4m 49s |  hbase-server generated 1 new + 0 unchanged - 0 fixed = 1 total (was 0)  |
||| _ Other Tests _ |
| +1 :green_heart: |  unit  |   1m 13s |  hbase-common in the patch passed.  |
| +1 :green_heart: |  unit  |  88m 35s |  hbase-server in the patch passed.  |
| -1 :x: |  asflicense  |   0m 48s |  The patch generated 1 ASF License warnings.  |
|  |   | 158m 18s |   |


| Reason | Tests |
|-------:|:------|
| FindBugs | module:hbase-server |
|  |  Dead store to posBeforeEncode in org.apache.hadoop.hbase.io.hfile.HFileBlock$Writer.write(Cell)  At HFileBlock.java:org.apache.hadoop.hbase.io.hfile.HFileBlock$Writer.write(Cell)  At HFileBlock.java:[line 914] |


| Subsystem | Report/Notes |
|----------:|:-------------|
| Docker | Client=19.03.6 Server=19.03.6 base: https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-1219/1/artifact/out/Dockerfile |
| GITHUB PR | https://github.com/apache/hbase/pull/1219 |
| Optional Tests | dupname asflicense javac javadoc unit spotbugs findbugs shadedjars hadoopcheck hbaseanti checkstyle compile |
| uname | Linux fe3d93846152 4.15.0-74-generic #84-Ubuntu SMP Thu Dec 19 08:06:28 UTC 2019 x86_64 GNU/Linux |
| Build tool | maven |
| Personality | /home/jenkins/jenkins-slave/workspace/Base-PreCommit-GitHub-PR_PR-1219/out/precommit/personality/provided.sh |
| git revision | master / 3c3aae9a2f |
| Default Java | 1.8.0_181 |
| shadedjars | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-1219/1/artifact/out/patch-shadedjars.txt |
| javadoc | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-1219/1/artifact/out/diff-javadoc-javadoc-hbase-common.txt |
| findbugs | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-1219/1/artifact/out/new-findbugs-hbase-server.html |
|  Test Results | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-1219/1/testReport/ |
| asflicense | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-1219/1/artifact/out/patch-asflicense-problems.txt |
| Max. process+thread count | 7544 (vs. ulimit of 10000) |
| modules | C: hbase-common hbase-server U: . |
| Console output | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-1219/1/console |
| versions | git=2.11.0 maven=2018-06-17T18:33:14Z) findbugs=3.1.11 |
| Powered by | Apache Yetus 0.11.1 https://yetus.apache.org |


This message was automatically generated.

**Comment by Apache-HBase** [comment]
:broken_heart: **-1 overall**






| Vote | Subsystem | Runtime | Comment |
|:----:|----------:|--------:|:--------|
| +0 :ok: |  reexec  |   1m 23s |  Docker mode activated.  |
||| _ Prechecks _ |
| +1 :green_heart: |  dupname  |   0m  0s |  No case conflicting files found.  |
| +1 :green_heart: |  hbaseanti  |   0m  0s |  Patch does not have any anti-patterns.  |
| +1 :green_heart: |  @author  |   0m  0s |  The patch does not contain any @author tags.  |
| +1 :green_heart: |  test4tests  |   0m  0s |  The patch appears to include 1 new or modified test files.  |
||| _ master Compile Tests _ |
| +0 :ok: |  mvndep  |   0m 34s |  Maven dependency ordering for branch  |
| +1 :green_heart: |  mvninstall  |   5m 39s |  master passed  |
| +1 :green_heart: |  compile  |   1m 21s |  master passed  |
| +1 :green_heart: |  checkstyle  |   1m 39s |  master passed  |
| +1 :green_heart: |  shadedjars  |   5m  5s |  branch has no errors when building our shaded downstream artifacts.  |
| +1 :green_heart: |  javadoc  |   0m 58s |  master passed  |
| +0 :ok: |  spotbugs  |   4m 43s |  Used deprecated FindBugs config; considering switching to SpotBugs.  |
| +1 :green_heart: |  findbugs  |   5m 30s |  master passed  |
||| _ Patch Compile Tests _ |
| +0 :ok: |  mvndep  |   0m 14s |  Maven dependency ordering for patch  |
| +1 :green_heart: |  mvninstall  |   5m 28s |  the patch passed  |
| +1 :green_heart: |  compile  |   1m 22s |  the patch passed  |
| +1 :green_heart: |  javac  |   1m 22s |  the patch passed  |
| +1 :green_heart: |  checkstyle  |   0m 23s |  The patch passed checkstyle in hbase-common  |
| +1 :green_heart: |  checkstyle  |   1m 15s |  hbase-server: The patch generated 0 new + 15 unchanged - 1 fixed = 15 total (was 16)  |
| +1 :green_heart: |  whitespace  |   0m  0s |  The patch has no whitespace issues.  |
| -1 :x: |  shadedjars  |   4m  2s |  patch has 10 errors when building our shaded downstream artifacts.  |
| +1 :green_heart: |  hadoopcheck  |  17m 28s |  Patch does not cause any errors with Hadoop 2.8.5 2.9.2 or 3.1.2.  |
| -1 :x: |  javadoc  |   0m 22s |  hbase-common generated 1 new + 0 unchanged - 0 fixed = 1 total (was 0)  |
| -1 :x: |  findbugs  |   5m  3s |  hbase-server generated 1 new + 0 unchanged - 0 fixed = 1 total (was 0)  |
||| _ Other Tests _ |
| +1 :green_heart: |  unit  |   1m 13s |  hbase-common in the patch passed.  |
| +1 :green_heart: |  unit  |  88m 16s |  hbase-server in the patch passed.  |
| -1 :x: |  asflicense  |   0m 47s |  The patch generated 1 ASF License warnings.  |
|  |   | 158m 47s |   |


| Reason | Tests |
|-------:|:------|
| FindBugs | module:hbase-server |
|  |  Dead store to posBeforeEncode in org.apache.hadoop.hbase.io.hfile.HFileBlock$Writer.write(Cell)  At HFileBlock.java:org.apache.hadoop.hbase.io.hfile.HFileBlock$Writer.write(Cell)  At HFileBlock.java:[line 914] |


| Subsystem | Report/Notes |
|----------:|:-------------|
| Docker | Client=19.03.6 Server=19.03.6 base: https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-1219/2/artifact/out/Dockerfile |
| GITHUB PR | https://github.com/apache/hbase/pull/1219 |
| Optional Tests | dupname asflicense javac javadoc unit spotbugs findbugs shadedjars hadoopcheck hbaseanti checkstyle compile |
| uname | Linux 2c67def89e01 4.15.0-74-generic #84-Ubuntu SMP Thu Dec 19 08:06:28 UTC 2019 x86_64 GNU/Linux |
| Build tool | maven |
| Personality | /home/jenkins/jenkins-slave/workspace/Base-PreCommit-GitHub-PR_PR-1219/out/precommit/personality/provided.sh |
| git revision | master / 3c3aae9a2f |
| Default Java | 1.8.0_181 |
| shadedjars | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-1219/2/artifact/out/patch-shadedjars.txt |
| javadoc | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-1219/2/artifact/out/diff-javadoc-javadoc-hbase-common.txt |
| findbugs | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-1219/2/artifact/out/new-findbugs-hbase-server.html |
|  Test Results | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-1219/2/testReport/ |
| asflicense | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-1219/2/artifact/out/patch-asflicense-problems.txt |
| Max. process+thread count | 6226 (vs. ulimit of 10000) |
| modules | C: hbase-common hbase-server U: . |
| Console output | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-1219/2/console |
| versions | git=2.11.0 maven=2018-06-17T18:33:14Z) findbugs=3.1.11 |
| Powered by | Apache Yetus 0.11.1 https://yetus.apache.org |


This message was automatically generated.

**Comment by Apache-HBase** [comment]
:confetti_ball: **+1 overall**






| Vote | Subsystem | Runtime | Comment |
|:----:|----------:|--------:|:--------|
| +0 :ok: |  reexec  |   0m 30s |  Docker mode activated.  |
||| _ Prechecks _ |
| +1 :green_heart: |  dupname  |   0m  0s |  No case conflicting files found.  |
| +1 :green_heart: |  hbaseanti  |   0m  0s |  Patch does not have any anti-patterns.  |
| +1 :green_heart: |  @author  |   0m  0s |  The patch does not contain any @author tags.  |
| +1 :green_heart: |  test4tests  |   0m  0s |  The patch appears to include 1 new or modified test files.  |
||| _ master Compile Tests _ |
| +0 :ok: |  mvndep  |   0m 37s |  Maven dependency ordering for branch  |
| +1 :green_heart: |  mvninstall  |   5m 49s |  master passed  |
| +1 :green_heart: |  compile  |   1m 32s |  master passed  |
| +1 :green_heart: |  checkstyle  |   1m 38s |  master passed  |
| +1 :green_heart: |  shadedjars  |   5m 20s |  branch has no errors when building our shaded downstream artifacts.  |
| +1 :green_heart: |  javadoc  |   1m  5s |  master passed  |
| +0 :ok: |  spotbugs  |   4m 17s |  Used deprecated FindBugs config; considering switching to SpotBugs.  |
| +1 :green_heart: |  findbugs  |   5m 11s |  master passed  |
||| _ Patch Compile Tests _ |
| +0 :ok: |  mvndep  |   0m 15s |  Maven dependency ordering for patch  |
| +1 :green_heart: |  mvninstall  |   5m 26s |  the patch passed  |
| +1 :green_heart: |  compile  |   1m 22s |  the patch passed  |
| +1 :green_heart: |  javac  |   1m 22s |  the patch passed  |
| +1 :green_heart: |  checkstyle  |   0m 24s |  The patch passed checkstyle in hbase-common  |
| +1 :green_heart: |  checkstyle  |   1m  8s |  hbase-server: The patch generated 0 new + 15 unchanged - 1 fixed = 15 total (was 16)  |
| +1 :green_heart: |  whitespace  |   0m  0s |  The patch has no whitespace issues.  |
| +1 :green_heart: |  shadedjars  |   4m 38s |  patch has no errors when building our shaded downstream artifacts.  |
| +1 :green_heart: |  hadoopcheck  |  16m  5s |  Patch does not cause any errors with Hadoop 2.8.5 2.9.2 or 3.1.2.  |
| +1 :green_heart: |  javadoc  |   1m  0s |  the patch passed  |
| +1 :green_heart: |  findbugs  |   5m 18s |  the patch passed  |
||| _ Other Tests _ |
| +1 :green_heart: |  unit  |   1m  4s |  hbase-common in the patch passed.  |
| +1 :green_heart: |  unit  |  61m  6s |  hbase-server in the patch passed.  |
| +1 :green_heart: |  asflicense  |   1m  6s |  The patch does not generate ASF License warnings.  |
|  |   | 127m 28s |   |


| Subsystem | Report/Notes |
|----------:|:-------------|
| Docker | Client=19.03.6 Server=19.03.6 base: https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-1219/3/artifact/out/Dockerfile |
| GITHUB PR | https://github.com/apache/hbase/pull/1219 |
| Optional Tests | dupname asflicense javac javadoc unit spotbugs findbugs shadedjars hadoopcheck hbaseanti checkstyle compile |
| uname | Linux 54e20a27078e 4.15.0-60-generic #67-Ubuntu SMP Thu Aug 22 16:55:30 UTC 2019 x86_64 GNU/Linux |
| Build tool | maven |
| Personality | /home/jenkins/jenkins-slave/workspace/Base-PreCommit-GitHub-PR_PR-1219/out/precommit/personality/provided.sh |
| git revision | master / 3c3aae9a2f |
| Default Java | 1.8.0_181 |
|  Test Results | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-1219/3/testReport/ |
| Max. process+thread count | 8851 (vs. ulimit of 10000) |
| modules | C: hbase-common hbase-server U: . |
| Console output | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-1219/3/console |
| versions | git=2.11.0 maven=2018-06-17T18:33:14Z) findbugs=3.1.11 |
| Powered by | Apache Yetus 0.11.1 https://yetus.apache.org |


This message was automatically generated.

File: hbase-common/src/main/java/org/apache/hadoop/hbase/io/encoding/DataBlockEncoder.java [hunk 1]
```diff
@@ -50,9 +50,8 @@ void startBlockEncoding(HFileBlockEncodingContext encodingCtx, DataOutputStream
 
   /**
    * Encodes a KeyValue.
-   * @return unencoded kv size written
```
Comment (anoopsjohn) [inline hbase-common/src/main/java/org/apache/hadoop/hbase/io/encoding/DataBlockEncoder.java]: In the Javadoc pls add a line which clearly says the need to call the EncodingState#postEncode() after the encoding step

**Comment by ramkrish86** [comment]
+1

**Comment by Apache-HBase** [comment]
:broken_heart: **-1 overall**






| Vote | Subsystem | Runtime | Comment |
|:----:|----------:|--------:|:--------|
| +0 :ok: |  reexec  |   2m 35s |  Docker mode activated.  |
||| _ Prechecks _ |
| +1 :green_heart: |  dupname  |   0m  1s |  No case conflicting files found.  |
| +1 :green_heart: |  hbaseanti  |   0m  0s |  Patch does not have any anti-patterns.  |
| +1 :green_heart: |  @author  |   0m  0s |  The patch does not contain any @author tags.  |
| +1 :green_heart: |  test4tests  |   0m  0s |  The patch appears to include 1 new or modified test files.  |
||| _ master Compile Tests _ |
| +0 :ok: |  mvndep  |   0m 35s |  Maven dependency ordering for branch  |
| +1 :green_heart: |  mvninstall  |   5m 47s |  master passed  |
| +1 :green_heart: |  compile  |   1m 21s |  master passed  |
| +1 :green_heart: |  checkstyle  |   1m 38s |  master passed  |
| +1 :green_heart: |  shadedjars  |   5m  9s |  branch has no errors when building our shaded downstream artifacts.  |
| +1 :green_heart: |  javadoc  |   0m 57s |  master passed  |
| +0 :ok: |  spotbugs  |   5m 37s |  Used deprecated FindBugs config; considering switching to SpotBugs.  |
| +1 :green_heart: |  findbugs  |   6m 26s |  master passed  |
||| _ Patch Compile Tests _ |
| +0 :ok: |  mvndep  |   0m 12s |  Maven dependency ordering for patch  |
| +1 :green_heart: |  mvninstall  |   5m 28s |  the patch passed  |
| +1 :green_heart: |  compile  |   1m 22s |  the patch passed  |
| +1 :green_heart: |  javac  |   1m 22s |  the patch passed  |
| +1 :green_heart: |  checkstyle  |   0m 23s |  The patch passed checkstyle in hbase-common  |
| +1 :green_heart: |  checkstyle  |   1m 14s |  hbase-server: The patch generated 0 new + 15 unchanged - 1 fixed = 15 total (was 16)  |
| +1 :green_heart: |  whitespace  |   0m  0s |  The patch has no whitespace issues.  |
| +1 :green_heart: |  shadedjars  |   5m  9s |  patch has no errors when building our shaded downstream artifacts.  |
| +1 :green_heart: |  hadoopcheck  |  17m 15s |  Patch does not cause any errors with Hadoop 2.8.5 2.9.2 or 3.1.2.  |
| +1 :green_heart: |  javadoc  |   0m 56s |  the patch passed  |
| +1 :green_heart: |  findbugs  |   6m 38s |  the patch passed  |
||| _ Other Tests _ |
| +1 :green_heart: |  unit  |   1m 13s |  hbase-common in the patch passed.  |
| -1 :x: |  unit  |  85m 16s |  hbase-server in the patch failed.  |
| +1 :green_heart: |  asflicense  |   1m  7s |  The patch does not generate ASF License warnings.  |
|  |   | 157m 51s |   |


| Reason | Tests |
|-------:|:------|
| Failed junit tests | hadoop.hbase.client.TestAsyncTableAdminApi |
|   | hadoop.hbase.client.TestAvoidCellReferencesIntoShippedBlocks |
|   | hadoop.hbase.client.TestAsyncClusterAdminApi2 |
|   | hadoop.hbase.client.TestAsyncNonMetaRegionLocatorConcurrenyLimit |
|   | hadoop.hbase.client.TestScannersFromClientSide2 |
|   | hadoop.hbase.client.TestMobCloneSnapshotFromClientError |


| Subsystem | Report/Notes |
|----------:|:-------------|
| Docker | Client=19.03.6 Server=19.03.6 base: https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-1219/4/artifact/out/Dockerfile |
| GITHUB PR | https://github.com/apache/hbase/pull/1219 |
| Optional Tests | dupname asflicense javac javadoc unit spotbugs findbugs shadedjars hadoopcheck hbaseanti checkstyle compile |
| uname | Linux d607b781dab4 4.15.0-74-generic #84-Ubuntu SMP Thu Dec 19 08:06:28 UTC 2019 x86_64 GNU/Linux |
| Build tool | maven |
| Personality | /home/jenkins/jenkins-slave/workspace/Base-PreCommit-GitHub-PR_PR-1219/out/precommit/personality/provided.sh |
| git revision | master / c21b28fc5c |
| Default Java | 1.8.0_181 |
| unit | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-1219/4/artifact/out/patch-unit-hbase-server.txt |
|  Test Results | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-1219/4/testReport/ |
| Max. process+thread count | 5840 (vs. ulimit of 10000) |
| modules | C: hbase-common hbase-server U: . |
| Console output | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-1219/4/console |
| versions | git=2.11.0 maven=2018-06-17T18:33:14Z) findbugs=3.1.11 |
| Powered by | Apache Yetus 0.11.1 https://yetus.apache.org |


This message was automatically generated.

**Comment by Apache-HBase** [comment]
:confetti_ball: **+1 overall**






| Vote | Subsystem | Runtime | Comment |
|:----:|----------:|--------:|:--------|
| +0 :ok: |  reexec  |   1m  5s |  Docker mode activated.  |
||| _ Prechecks _ |
| +1 :green_heart: |  dupname  |   0m  0s |  No case conflicting files found.  |
| +1 :green_heart: |  hbaseanti  |   0m  0s |  Patch does not have any anti-patterns.  |
| +1 :green_heart: |  @author  |   0m  0s |  The patch does not contain any @author tags.  |
| +1 :green_heart: |  test4tests  |   0m  0s |  The patch appears to include 1 new or modified test files.  |
||| _ master Compile Tests _ |
| +0 :ok: |  mvndep  |   0m 35s |  Maven dependency ordering for branch  |
| +1 :green_heart: |  mvninstall  |   5m 46s |  master passed  |
| +1 :green_heart: |  compile  |   1m 22s |  master passed  |
| +1 :green_heart: |  checkstyle  |   1m 37s |  master passed  |
| +1 :green_heart: |  shadedjars  |   5m  9s |  branch has no errors when building our shaded downstream artifacts.  |
| +1 :green_heart: |  javadoc  |   0m 58s |  master passed  |
| +0 :ok: |  spotbugs  |   5m 37s |  Used deprecated FindBugs config; considering switching to SpotBugs.  |
| +1 :green_heart: |  findbugs  |   6m 26s |  master passed  |
||| _ Patch Compile Tests _ |
| +0 :ok: |  mvndep  |   0m 13s |  Maven dependency ordering for patch  |
| +1 :green_heart: |  mvninstall  |   5m 34s |  the patch passed  |
| +1 :green_heart: |  compile  |   1m 23s |  the patch passed  |
| +1 :green_heart: |  javac  |   1m 23s |  the patch passed  |
| +1 :green_heart: |  checkstyle  |   0m 23s |  The patch passed checkstyle in hbase-common  |
| +1 :green_heart: |  checkstyle  |   1m 12s |  hbase-server: The patch generated 0 new + 15 unchanged - 1 fixed = 15 total (was 16)  |
| +1 :green_heart: |  whitespace  |   0m  0s |  The patch has no whitespace issues.  |
| +1 :green_heart: |  shadedjars  |   5m 11s |  patch has no errors when building our shaded downstream artifacts.  |
| +1 :green_heart: |  hadoopcheck  |  17m 35s |  Patch does not cause any errors with Hadoop 2.8.5 2.9.2 or 3.1.2.  |
| +1 :green_heart: |  javadoc  |   1m  4s |  the patch passed  |
| +1 :green_heart: |  findbugs  |   7m  5s |  the patch passed  |
||| _ Other Tests _ |
| +1 :green_heart: |  unit  |   1m 19s |  hbase-common in the patch passed.  |
| +1 :green_heart: |  unit  | 100m 17s |  hbase-server in the patch passed.  |
| +1 :green_heart: |  asflicense  |   0m 47s |  The patch does not generate ASF License warnings.  |
|  |   | 172m 20s |   |


| Subsystem | Report/Notes |
|----------:|:-------------|
| Docker | Client=19.03.6 Server=19.03.6 base: https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-1219/5/artifact/out/Dockerfile |
| GITHUB PR | https://github.com/apache/hbase/pull/1219 |
| Optional Tests | dupname asflicense javac javadoc unit spotbugs findbugs shadedjars hadoopcheck hbaseanti checkstyle compile |
| uname | Linux 4912d40edb1a 4.15.0-74-generic #84-Ubuntu SMP Thu Dec 19 08:06:28 UTC 2019 x86_64 GNU/Linux |
| Build tool | maven |
| Personality | /home/jenkins/jenkins-slave/workspace/Base-PreCommit-GitHub-PR_PR-1219/out/precommit/personality/provided.sh |
| git revision | master / c0301e3fdf |
| Default Java | 1.8.0_181 |
|  Test Results | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-1219/5/testReport/ |
| Max. process+thread count | 5879 (vs. ulimit of 10000) |
| modules | C: hbase-common hbase-server U: . |
| Console output | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-1219/5/console |
| versions | git=2.11.0 maven=2018-06-17T18:33:14Z) findbugs=3.1.11 |
| Powered by | Apache Yetus 0.11.1 https://yetus.apache.org |


This message was automatically generated.

**Pull request merged**